src/main/java/org/jspecify/nullness/Nullable.java

```diff
@@ -46,124 +47,61 @@
- *   <li>On a non-wildcard<b> type argument:</b> A type usage of "nullable string" appears
- *       <i>within</i> the compound type {@code List<@Nullable String>}. No matter how this type is
- *       used (return type, local variable type, etc.), this tells us the same thing: when using
- *       this type, every appearance of {@code E} in {@code List}'s member signatures will be
- *       considered nullable. For a list, this means it may contain null <i>elements</i>. If the
- *       list reference itself might be null as well, we can write {@code @Nullable List<@Nullable
- *       String>}, a "nullable list of nullable strings". Type arguments supplied to a generic
- *       method invocation or class instance creation expression work similarly.
- *   <li>On a<b> type parameter bound:</b> If a type parameter's upper bound is annotated with
- *       {@code @Nullable}, as in {@code class Foo<T extends @Nullable Object>}, that means that
- *       {@code T} may be bound to nullable types, as in {@code Foo<@Nullable String>}.
- *   <li>On a<b> type variable</b> usage: A type parameter, like the {@code E} in {@code interface
- *       List<E>}, defines a type variable of the same name, usable only <i>within</i> the scope of
- *       the declaring API element. In any example using {@code String} above, a type variable like
- *       {@code E} might appear instead. {@code @Nullable} continues to mean "or null" as always,
- *       but notably, this works without regard to whether the type argument is <i>already</i>
- *       nullable. For example, given {@code class Foo<E extends @Nullable Object>}, with a method
- *       {@code @Nullable E eOrNull()}, then whether {@code foo} is of type {@code Foo<String>} or
- *       {@code Foo<@Nullable String>}, the expression {@code foo.eOrNull()} is always nullable.
- *       Using {@code @Nullable E} in this way is called "nullable projection" (<a
- *       href="NonNull.html#projection">non-null projection</a> is likewise supported, but less
- *       commonly useful).
+ *   <li>On a <b>type argument</b>: A type usage of "nullable string" appears <i>within</i> the
+ *       compound type {@code List<@Nullable String>}. No matter how this type is used (return type,
+ *       etc.), this means the same thing: every appearance of {@code E} in {@code List}'s member
+ *       signatures will be considered nullable. For a list, this means it may contain null
+ *       <i>elements</i>. If the list reference itself might be null as well, we can write
+ *       {@code @Nullable List<@Nullable String>}, a "nullable list of nullable strings".
+ *   <li>On the upper bound of a <b>type parameter</b>: For example, as seen in {@code class List<E
+ *       extends @Nullable Object>}. This means that a <i>type argument</i> supplied for that type
+ *       parameter is permitted to be nullable if desired: {@code List<@Nullable String>}. A
```

Comment:
Maybe cut "if desired?"

Reply:
If this is a "maybe" I'll lean toward keeping it for emphasis.